pkg/scheduler/apis/config/loader/loader.go

```diff
@@ -24,27 +24,13 @@ import (
 	"github.com/gardener/gardener/pkg/scheduler/apis/config"
 	"github.com/gardener/gardener/pkg/scheduler/apis/config/install"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/apimachinery/pkg/runtime/serializer/json"
-	"k8s.io/apimachinery/pkg/runtime/serializer/versioning"
 )
 
-var (
-	Codec  runtime.Codec
-	Scheme *runtime.Scheme
-)
+var Scheme *runtime.Scheme
 
 func init() {
 	Scheme = scheme.Scheme
 	install.Install(Scheme)
```

Comment:
This also installs the config API types into the kubernetes `scheme.Scheme`. Can you fix that?